applications/crossbar/doc/search.md

```diff
@@ -39,3 +47,35 @@ The Search API allows queries on databases.
     }
 
 
+### _GET_ - Multi Search
+
+    t = document type
+    view_name = value
+
+
+    curl -v -X GET -H "X-Auth-Token: {AUTH_TOKEN}" 'http://localhost:8000/v2/search?t=account&name=test&realm=test'
+
+
+```
+{
+  "data": {
+    "realm": [],
```

Comment:
Why are `realm` and `name` on the response if they aren't populated with anything? Shouldn't `data` just be the summary listing of the search results (similar to `GET /v2/accounts/{ACCOUNT_ID}/descendants` and the like?)